### PR TITLE
Add Polymesh to other chains

### PR DESCRIPTION
**Pull request by F-OBrien**

### CODE REVIEW

**Comment by yashovardhan** [comment]
@rtomas please have a look. Also update for v8, rebase and fix merge conflicts

File: docs/connect-blockchain/other/polymesh.mdx [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+>
+
+```typescript
+const chainConfig = {
+  chainNamespace: "other",
+  chainId: "Polymesh", //
+  rpcTarget: "https://mainnet-rpc.polymesh.network",
+  // Avoid using public rpcTarget in production.
+  displayName: "Polymesh Mainnet",
+  blockExplorer: "https://polymesh.subscan.io",
```
Comment (rtomas) [inline docs/connect-blockchain/other/polymesh.mdx:50]: in v8 is called -> blockExplorerUrl

**Review by rtomas — APPROVED** [review]
it's ready to merge

**Comment by rtomas** [comment]
@yashovardhan I need someone to approve to run the Vercel Action. Thanks :)

**Pull request merged**